src/talker.h

```diff
@@ -90,6 +90,9 @@ class talker
             return 0;
         }
         virtual tripoint pos() const = 0;
+        virtual tripoint_abs_ms global_pos() const {
```

Comment:
If this function can not return any meaningful value, it should be made virtual.
```suggestion
        virtual tripoint_abs_ms global_pos() const = 0;
```